pkg/util/util.go

```diff
@@ -989,6 +989,18 @@ func DownloadFile(url string, filepath string) error {
 	return nil
 }
 
+// DownloadFileInMemory uses the url to download the file and return bytes
+func DownloadFileInMemory(url string) ([]byte, error) {
```

Comment:
Call this from `DownloadFile` and make sure that the HTTP client is created consistently in both functions (which isn't the case in this instance).

Reply:
Done